dll/cpl/inetcpl/security.c

```diff
@@ -275,10 +275,6 @@ static INT_PTR security_on_destroy(secdlg_data * sd)
 
     heap_free(sd->zone_attr);
     heap_free(sd->zones);
-    if (sd->himages) {
-        SendMessageW(sd->hlv, LVM_SETIMAGELIST, LVSIL_NORMAL, 0);
-        ImageList_Destroy(sd->himages);
-    }
```

Comment:
Ah yes, so what I suggested was, to keep that code, but to surround it with `#ifndef __REACTOS__  ... #endif` since this is wine-synced code.

Reply:
Fixed.